pwndbg/gdblib/typeinfo.py

```diff
@@ -25,11 +25,22 @@ def lookup_types(*types):
 
 
 def update():
+    # Workaround for Rust stuff, see https://github.com/pwndbg/pwndbg/issues/855
+    lang = gdb.execute("show language", to_string=True)
+    if "rust" not in lang:
+        restore_lang = None
+    else:
```

Comment:
This is fine, but you may prefer this:
```python
restore_lang = None:
if "rust" in lang:
    gdb.execute("set language c")
    if '"auto;' in lang:
        restore_lang = "auto"
    else:
        restore_lang = "rust"
```

Reply:
Given that it is executed on many events, lets leave it as it is
(not that this is bottleneck but still ;P)